dashboard/common/db_util.py

```diff
@@ -102,9 +102,13 @@ def get_user_courses_info(username):
             cursor.execute("select canvas_id, name from course where canvas_id in %s",[course_tuple])
             course_names = cursor.fetchall()
             df = pd.DataFrame(list(course_names))
-            df.columns = ["course_id", "course_name"]
-            course_info = df.to_dict(orient='records')
-            logger.info(f"User {username} is enrolled in these courses: {course_info}")
+
+            # create an empty dict
+            course_info = dict()
```

Comment:
It looks like course_info here is already set up as an empty list so these lines aren't needed. I tested without it and it works fine. 

The expected result from this is a list of objects
`[{column -> value}, … , {column -> value}]
`
So an empty list is expected.